.changeset/spotty-bears-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': minor
+---
+
+Change version bump CLI command to use common HTTP_PROXY env variables
```

Comment:
Would it be possible to not make this a breaking change given that this has been the established pattern for some time now? I think the overall idea of this PR is good though 👍